plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -248,7 +243,11 @@ def _get_cli_vars(self) -> str:
             (self.templater_selector, self.name, "context")
         )
 
-        return str(cli_vars) if cli_vars else "{}"
+        return cli_vars if cli_vars else dict()
+
+    def _get_threads(self) -> int:
```

Comment:
What is the `threads` parameter to dbt? I feel uncomfortable reusing the SQLFluff concurrent linting setting for something that looks different (processes vs threads, SQLFluff linting vs ???).

Reply:
Seems like the whole function could be removed, perhaps?